game-core/src/main/java/games/strategy/engine/lobby/client/login/LobbyLoginPreferences.java

```diff
@@ -77,7 +79,7 @@ private static Credentials unprotectCredentials(
           credentials.password.isEmpty() ? "" : credentialManager.unprotectToString(credentials.password),
           false);
     } catch (final CredentialManagerException e) {
-      ClientLogger.logQuietly("failed to unprotect lobby login credentials", e);
+      log.log(Level.SEVERE, "failed to unprotect lobby login credentials", e);
```

Comment:
`WARNING` level is probably more appropriate for the two cases in this file because the condition is completely recoverable (the user just has to re-enter their credentials).

Reply:
The error message should say that " user just has to re-enter their credentials", or even do it as an info message to them.

I'm going by the assumption that these messages are never seen by users, after a decade plus of these error messages never being an issue, we can remove them by default, they are silent debug that is never reported back to us.

This one will need verification of the UX to some extent, the 'safe' thing to do is just remove the logging as it did "in effect" nothing for users (who do not by habit play with the console window open).

Reply:
> WARNING level is probably more appropriate for the two cases in this file because the condition is completely recoverable (the user just has to re-enter their credentials).

We should consider this from a user perspective and not a java dev perspective : )

A user is going to see this error and likely save the game and restart, regardless if we call it 'warning/severe'. I think at the moment there is no substantial difference between warning/severe, it's going to be the same for users - let's define a UX to correspond with different logging levels outside of this PR. Soon at least that will be a sane question compared to the hodge of logging options that we are leaving behind.